test/commands/upgrade/upgrade.test.ts

```diff
@@ -12,180 +12,181 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-import { ux } from '@oclif/core';
 import { expect, test } from '@oclif/test';
 import * as gitModule from 'simple-git';
-import * as packageModule from '../../../src/modules/package';
-import { ProjectConfigOptions } from '../../../src/modules/project-config';
-import { CliFileSystem } from '../../../src/utils/fs-bridge';
+// eslint-disable-next-line @typescript-eslint/naming-convention
+import Init from '../../../src/commands/init';
+import * as upgrade from '../../../src/commands/upgrade';
 import { simpleGitInstanceMock } from '../../helpers/simpleGit';
-import { velocitasConfigMock } from '../../utils/mockConfig';
-import { mockFolders, userHomeDir } from '../../utils/mockfs';
+import { installedCorePackage, installedRuntimePackage, installedSetupPackage, mockFolders } from '../../utils/mockfs';
 
 const mockedNewVersionTag = 'v2.0.0';
+const mockedLowerVersionTag = 'v1.0.0';
+const mockedHigherVersionTag = 'v1.1.2';
 
-describe('upgrade', () => {
-    test.do(() => {
-        mockFolders({ velocitasConfig: true });
-    })
-        .stdout()
-        .stub(gitModule, 'simpleGit', (stub) => stub.returns(simpleGitInstanceMock()))
-        .command(['upgrade', '--dry-run'])
-        .it('checking for upgrades in dry-run - no installed sources found', (ctx) => {
-            expect(ctx.stdout).to.contain('Checking for updates!');
-            expect(ctx.stdout).to.contain(
-                `... No installed sources for ${velocitasConfigMock.packages[0].repo}:${velocitasConfigMock.packages[0].version} found`,
-            );
-            expect(ctx.stdout).to.contain(
-                `... No installed sources for ${velocitasConfigMock.packages[1].repo}:${velocitasConfigMock.packages[1].version} found`,
-            );
-        });
+describe('upgrade command', () => {
+    describe('upgrade --dry-run', () => {
```

Comment:
We should think about readable sentences for our tests:
`<upgrade command> <with parameter --dry-run> <throws when no lock file is present>`